draft-ietf-quic-qpack.md

```diff
@@ -1355,14 +1357,14 @@ A decoder can limit the amount of state memory used for the dynamic table by
 setting an appropriate value for the maximum size of the dynamic table. In
 HTTP/3, this is realized by setting an appropriate value for the
 SETTINGS_QPACK_MAX_TABLE_CAPACITY parameter. An encoder can limit the amount of
-state memory it uses by signaling a lower dynamic table size than the decoder
-allows (see {{eviction}}).
+state memory it uses and signal a lower dynamic table size than the decoder
+allows to propagate that fact to the decoder (see {{eviction}}).
```

Comment:
Suggestion:

 An encoder can limit the amount of state memory it uses by choosing a smaller dynamic table size than the decoder allows, and signaling this to the decoder (see {{set-dynamic-table-capacity}}).

Reply:
Thanks; I pushed that with a slight modification (the signalling is not part of the act of limiting the encoder's state memory, but rather a related but separate act).

Reply:
Sorry I missed this comment earlier and basically reverted to my original text.  While the acts themselves are separate, the second is directly tied to the first -- the encoder can't change the size it is using without also signaling the decoder.